lib/plugin.js

```diff
@@ -62,8 +63,12 @@ function assignExperiment(ctx) {
       variantIndexes.push(index)
     }
 
-    // Write exp cookie if changed
     const expCookie = experiment.experimentID + '.' + variantIndexes.join('-')
+
+    // expose raw optimize token directly
+    experiment.token = expCookie
```

Comment:
Maybe can be merged with L66 like `experiment.token = experiment.experimentID + '.' + variantIndexes.join('-') ` ? to avoid extra `expCookie` variable?